lib/CL/pocl_llvm_build.cc

```diff
@@ -413,7 +413,7 @@ int pocl_llvm_build_program(cl_program program,
   size_t fastmath_flag = user_options.find("-cl-fast-relaxed-math");
 
   if (fastmath_flag != std::string::npos) {
-#ifdef ENABLE_CONFORMANCE
+#if defined(ENABLE_CONFORMANCE) && LLVM_VERSION_MAJOR < 18
```

Comment:
just a consistency nitpick: the macro we use in most of PoCL code is `LLVM_MAJOR`

Reply:
OK. I can fix it in a future commit to avoid retesting for this?

Reply:
Maybe we should convert all of those LLVM_VERSION_MAJOR references in a single commit.

Reply:
Sure, no problem